atk/src/lib.rs

```diff
@@ -7,11 +7,11 @@
 //! This library contains safe Rust bindings for [ATK](https://developer.gnome.org/atk/). It's
 //! a part of [Gtk-rs](https://gtk-rs.org/).
 
-#![cfg_attr(feature = "cargo-clippy", allow(let_unit_value))]
-#![cfg_attr(feature = "cargo-clippy", allow(new_without_default))]
-#![cfg_attr(feature = "cargo-clippy", allow(type_complexity))]
-#![cfg_attr(feature = "cargo-clippy", allow(trivially_copy_pass_by_ref))]
-#![cfg_attr(feature = "cargo-clippy", allow(derive_hash_xor_eq))]
+#![allow(clippy::let_unit_value)]
+#![allow(clippy::new_without_default)]
+#![allow(clippy::type_complexity)]
+#![allow(clippy::trivially_copy_pass_by_ref)]
+#![allow(clippy::derive_hash_xor_eq)]
```

Comment:
We should distinguish here by things in `auto` and elsewhere. In non-auto I would prefer to get them fixed or per case to add an `#[allow]` to it.

Reply:
It seems atk doesn't need any of those so I've removed them

Reply:
I will do the same for other crates though.

Reply:
Should be done